credoai/utils/model_utils.py

```diff
@@ -1,11 +1,15 @@
 import warnings
-from credoai.utils import global_logger
 
 from sklearn.base import is_classifier, is_regressor
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.utils import multiclass
 
-from tensorflow.keras import layers
+from credoai.utils import global_logger
```

Comment:
I don't think this file needs the "if tf_exists" paradigm in the `validate_keras_clf` function since we'd only be calling that function if keras is successfully imported elsewhere. That said, it might be good to put it in anyway.